io.github.guillaumechereau.Goxel.json

```diff
@@ -0,0 +1,98 @@
+{
+  "id": "io.github.guillaumechereau.Goxel",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "goxel",
+  "rename-desktop-file": "goxel.desktop",
+  "rename-icon": "goxel",
+  "rename-appdata-file": "goxel.appdata.xml",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=home"
+  ],
+  "cleanup": [ "/lib/cmake" ],
+  "modules": [
+    {
+      "name": "scons",
+      "cleanup": [ "*" ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python setup.py install --prefix=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://downloads.sourceforge.net/project/scons/scons/2.5.1/scons-2.5.1.tar.gz",
+          "sha256": "0b25218ae7b46a967db42f2a53721645b3d42874a65f9552ad16ce26d30f51f2"
+        }
+      ]
+    },
+    {
+        "name": "glfw",
+        "buildsystem": "cmake-ninja",
+        "builddir": true,
+        "config-opts": [
+            "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+            "-DBUILD_SHARED_LIBS:BOOL=ON"
+        ],
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://github.com/glfw/glfw/releases/download/3.2.1/glfw-3.2.1.zip",
+                "sha256": "b7d55e13e07095119e7d5f6792586dd0849c9fcdd867d49a4a5ac31f982f7326"
+            }
+        ],
+        "cleanup": [
+            "/include",
+            "/lib/pkgconfig"
+        ]
+    },
+    {
+      "name": "imagemagick",
```

Comment:
This is only built to resize images or is it required for something else?

Reply:
Only for resizing the icon

Reply:
Personally I think thats a bit unnecessary of a build dep and would rather just throw the made images in git and save a the world some wasted electricity.